wpiunits/src/main/java/edu/wpi/first/units/Units.java

```diff
@@ -250,6 +250,43 @@ private Units() {
   public static final Mult<Mult<Mass, Distance>, Distance> KilogramSquareMeters =
       Kilograms.mult(Meters).mult(Meters);
 
+  // Force
+  /** The base SI unit for force. */
+  public static final Mult<Mass, Velocity<Velocity<Distance>>> Newtons =
+      derive(Kilograms.mult(Meters.per(Second).per(Second))).named("Newtons").symbol("N").make();
+
+  /** Singular alias for Newtons. */
+  public static final Mult<Mass, Velocity<Velocity<Distance>>> Newton = Newtons; // alias;
+
+  /** A unit of force equivalent to 4.448222 {@link #Newtons}. */
+  public static final Mult<Mass, Velocity<Velocity<Distance>>> PoundsForce =
+      derive(Newtons).named("Pounds-Force").symbol("lb.").make();
```

Comment:
This (and the other derived units) needs an aggregate or splitInto conversion factor

Reply:
woops.  Should be fixed now.